commands/databases.go

```diff
@@ -243,9 +245,44 @@ func buildDatabaseCreateRequestFromArgs(c *CmdConfig) (*godo.DatabaseCreateReque
 	}
 	r.PrivateNetworkUUID = privateNetworkUUID
 
+	restoreFromCluster, err := c.Doit.GetString(c.NS, doctl.ArgDatabaseRestoreFromCluster)
+	if err != nil {
+		return nil, err
+	}
+	if restoreFromCluster != "" {
+		backUpRestore := &godo.DatabaseBackupRestore{}
+		backUpRestore.DatabaseName = restoreFromCluster
+		// only set the restore-from-timestamp if restore-from-cluster is set.
+		restoreFromTimestamp, err := c.Doit.GetString(c.NS, doctl.ArgDatabaseRestoreFromTimestamp)
+		if err != nil {
+			return nil, err
+		}
+		if restoreFromTimestamp != "" {
+			dateFormatted, err := convertUTCtoISO8601(restoreFromTimestamp)
+			if err != nil {
+				return nil, err
+			}
+			backUpRestore.BackupCreatedAt = dateFormatted
+		}
+		r.BackupRestore = backUpRestore
+	}
+
+	r.PrivateNetworkUUID = privateNetworkUUID
+
 	return r, nil
 }
 
+func convertUTCtoISO8601(restoreFromTimestamp string) (string, error) {
+	// accepts UTC time format from user (to match db list output) and converts it to ISO8601 for api parity.
+	date, error := time.Parse("2006-01-02 15:04:05 +0000 UTC", restoreFromTimestamp)
```

Comment:
So just for consistency, this can use the `RFC822Z` const format since the time.UTC() method just uses that format and converts the given time to UTC timezone.

```suggestion
	date, error := time.Parse(time.RFC822Z, restoreFromTimestamp)
```

Reply:
I was reading in a stackoverflow thread that one need to use that specific string date to parse the correct date, as seen below:

Using the `2006-01-02 15:04:05 +0000 UTC` string in `time.Parse()`:
```
	dateString := "2023-02-01 17:32:15 +0000 UTC"
	date, _ := time.Parse("2006-01-02 15:04:05 +0000 UTC", dateString)
	dateFormatted := date.Format(time.RFC3339)

	fmt.Println("The time converted to iso format ", dateFormatted)
```
outputs: 
`The time converted to iso format  2023-02-01T17:32:15Z`


Using the `time.RFC822Z` string in `time.Parse()`:
```
	dateString := "2023-02-01 17:32:15 +0000 UTC"
	date, _ := time.Parse(time.RFC822Z, dateString)
	dateFormatted := date.Format(time.RFC3339)

	fmt.Println("The time converted to iso format ", dateFormatted)
```
outputs: 
`The time converted to iso format  0001-01-01T00:00:00Z`

🤷

Reply:
😮 nice catch.